### PR TITLE
Add: new meeting notes

### LINKED ISSUE #92: 20190729 - meeting
Just an hour hanging out and talking about DAT stuff.

Time: 29 AUG 2019, 18:00 GMT

https://zoom.us/j/181036238

### PR DESCRIPTION
**Pull request by dpaez**
This PR adds the notes of the last comm-comm meeting.

Thanks to @RangerMauve for the note-taking. 👏  

Thanks to everyone who joined!

Closes #92

### CODE REVIEW

File: meeting-notes/33-29Aug2019.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+29 Aug 2019
+
+## People
+
+* [Diego](https://github.com/dpaez) [[GEUT](https://geutstudio.com)]
+* [Mauve](https://github.com/rangermauve) [[Dat](datproject)]
+* [Paul Frazee](https://github.com/beakerbrowser/)
+* [Edward Silverton](https://github.com/edsilv)
+* Joshua
```
Comment (RangerMauve) [inline meeting-notes/33-29Aug2019.md]:
```suggestion
* [Joshua Mir](https://twitter.com/Jam10o)
```

File: meeting-notes/33-29Aug2019.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+* Swapnil
+* Martin Heideggar
```
Comment (RangerMauve) [inline meeting-notes/33-29Aug2019.md]:
```suggestion
* [Martin Heidegger](https://github.com/martinheidegger)
```

**Comment by RangerMauve** [comment]
Thanks for adding this in! 💜

**Comment by todrobbins** [comment]
LGTM!

**Pull request merged**